changelog/6397.fixed.md

```diff
@@ -0,0 +1 @@
+Allow optional node kind value
```

Comment:
It’s a bit too vague, let’s make it explicit that it's a bugfix that allow updating  the node kind to null in webhook